src/SystemCommands-MethodCommands/SycCopyTestMethodNameToClypboardCommand.class.st

```diff
@@ -0,0 +1,33 @@
+"
+Copy selected methods selector into Clipboard as testOriginalSelector
+"
+Class {
+	#name : #SycCopyTestMethodNameToClypboardCommand,
+	#superclass : #SycMethodCommand,
+	#category : #'SystemCommands-MethodCommands'
+}
+
+{ #category : #accessing }
+SycCopyTestMethodNameToClypboardCommand >> defaultMenuIconName [
+	^#smallCopy
+]
+
+{ #category : #accessing }
+SycCopyTestMethodNameToClypboardCommand >> defaultMenuItemName [
+	^'Copy test method selector(s) to Clipboard'
+]
+
+{ #category : #accessing }
+SycCopyTestMethodNameToClypboardCommand >> description [
+	^'Copy selected methods selector into Clipboard as testXXX'
+]
+
+{ #category : #execution }
+SycCopyTestMethodNameToClypboardCommand >> execute [
```

Comment:
There is a typo here. It sould be Clip, not Clyp.

Reply:
hehe, it is your typo because I just copied the class SycCopyMethodNameToClypboardCommand